README.md

```diff
@@ -1,175 +1,404 @@
-# postgres-mcp: PostgreSQL Tuning and Analysis Tool
+# Postgres Pro | MCP Server
```

Comment:
I was a little confused by the pipe. "Postgres Pro MCP Server" seems better. Also, seeing "Pro" confused me because it made me think that it was a paid service. Naming is hard (it really is), so I'm not sure what else to do, but I'm not wild about "Pro".

Also, let's lead with the bold statement that (something like) "This is the Postgres MCP server that should beat all others" or something communicating that this isn't just "some or another PG MCP server" but that it designed to be THE PG MCP server. My brain is tired so I'm not coming up with the right wording, but let's communicate our intent.